firefox-ios/Client/Application/AccessibilityIdentifiers.swift

```diff
@@ -71,6 +71,27 @@ public struct AccessibilityIdentifiers {
         static let actionButton = "ContextualHints.ActionButton"
     }
 
+    struct EnhancedTrackingProtection {
+        struct MainScreen {
+            static let clearCookiesButton = "TrackingProtection.ClearCookiesButton"
```

Comment:
This seems odd, the name should reflect the struct. 
It goes for all variables so `clearCookiesButton` should have value like `EnhancedTrackingProtection.MainScreen...`

Look at here for an example

https://github.com/mozilla-mobile/firefox-ios/pull/20533/files#diff-e28fbf5252641812fc303aac7fd5227f2471186cd8320232b7d390abac21b38dR181